neqo-transport/src/connection.rs

```diff
@@ -1219,7 +1227,7 @@ impl Connection {
 
             // Add frames to the packet.
             let (tokens, ack_eliciting) = if *space >= min_pn_space {
-                let r = self.add_frames(&mut builder, *space, limit, now);
+                let r = self.add_frames(&mut builder, *space, limit, only_not_cc, now);
                 if builder.is_empty() {
                     if pto {
                         // Add a PING if there is a PTO and nothing to send.
```

Comment:
Somewhere down there, it might pay to have some debug_assert!() calls to cross-verify cc_limited and pto and cc_limited and the available space.